jwst/ami/tests/test_ami_analyze.py

```diff
@@ -46,16 +43,9 @@ def test_ami_analyze_step_no_affine(example_model):
     AmiAnalyzeStep.call(example_model, affine2d=None)
 
 
-def create_throughput(nelem):
```

Comment:
whoops I should have deleted this a long time ago when I added all the conftest methods